src/driver-hidpp20.c

```diff
@@ -859,7 +859,8 @@ hidpp20drv_read_resolution_dpi(struct ratbag_profile *profile)
 	unsigned int default_rate = 500;
 	int rc;
 
-	if (drv_data->capabilities & HIDPP_CAP_RESOLUTION_2200) {
+	if (drv_data->capabilities & HIDPP_CAP_RESOLUTION_2200 &&
```

Comment:
doesn't the compiler complain about this? either way, please add `()` around this bit too.

Reply:
Not really, it will do the & first and then the &&. Either way, I added the parenthesis.

Reply:
weird, I was certain that the compiler warned about this. Anyway, just fwiw, this is a general style issue because the parens make the code immediately obvious and there's less chance of a bug being introduced. Technically not needed, humanly it usually is :)